cmake/update_pot_source_dependencies.cmake

```
@@ -35,6 +35,15 @@ if(DOMAIN STREQUAL ${DEFAULT_DOMAIN})
 
 						echo $$file >>
 							${PROJECT_SOURCE_DIR}/po/${DOMAIN}/POTFILES.in \;
+
+					# While files don't need a GETTEXT_DOMAIN to be included in
+					# the default domain, accept files that contain a
+					# GETTEXT_DOMAIN definition for the default domain too.
+					elif grep \"^\#define  *GETTEXT_DOMAIN  *\\\"${DOMAIN}\\\"\"
```

```suggestion
					elif grep '^#define  *GETTEXT_DOMAIN  *\"${DOMAIN}\"'
```
It seems to me like this should work as well and is more readable.

---

The backslash before the `#` was definitely necessary, but I took the rest of the suggestion.

---

Ah, thanks for checking. Must be cmake syntax then since neither shell nor grep care.